common/src/sql/dbinit.sql

```diff
@@ -862,10 +863,43 @@ CREATE UNIQUE INDEX ON omicron.public.instance (
 -- Allow looking up instances by server. This is particularly
 -- useful for resource accounting within a sled.
 CREATE INDEX ON omicron.public.instance (
-    active_server_id
+    active_sled_id
 ) WHERE
     time_deleted IS NULL;
 
+/*
+ * A special view of an instance provided to operators for insights into what's running 
+ * on a sled.
+ */
+
+CREATE VIEW omicron.public.sled_instance 
+AS SELECT
+   instance.id,
+   instance.name,
+   silo.name as silo_name,
+   project.name as project_name,
+   instance.active_sled_id,
+   instance.time_created,
+   instance.time_modified,
+   instance.migration_id,
+   instance.ncpus,
+   instance.memory,
+   instance.state
+FROM
+    omicron.public.instance as instance
+JOIN
+    omicron.public.project as project
+ON  
+    instance.project_id = project.id
+JOIN
+    omicron.public.silo as silo
+ON  
+    project.silo_id = silo.id
+WHERE
+    instance.active_sled_id IS NOT NULL
```

Comment:
I'm a little inclined to say just remove this condition. I don't think this is ever `NULL` today once the instance has been created but there's an issue (#2315) to actually make it `Nullable/Option`. But even at then, `DataStore::sled_instance_list` already adds a filter `active_sled_id` matches the given `authz_sled`.